superset/config.py

```diff
@@ -1735,6 +1735,11 @@ def EMAIL_HEADER_MUTATOR(  # pylint: disable=invalid-name,unused-argument  # noq
 SLACK_PROXY = None
 SLACK_CACHE_TIMEOUT = int(timedelta(days=1).total_seconds())
 
+# Maximum number of retries when Slack API returns rate limit errors
+# Default: 5
+# For workspaces with 10k+ channels, consider increasing to 10
+SLACK_API_RATE_LIMIT_RETRY_COUNT = 5
```

Comment:
```suggestion
SLACK_API_RATE_LIMIT_RETRY_COUNT = 2
```
Place at 2 to match the existing code behavior. Allow others to update if they need more than 2.